pkgs/desktops/gnome-3/core/tracker-miners/default.nix

```diff
@@ -51,6 +84,18 @@ in stdenv.mkDerivation rec {
     };
   };
 
+  postPatch = ''
+    substituteInPlace src/tracker-extract/10-flac.rule --replace audio/x-flac audio/flac
+    substituteInPlace src/tracker-extract/meson.build \
+      --replace tracker_common_dep tracker_miners_common_dep
+  '';
+
+  postFixup = ''
+    ${glib.dev}/bin/glib-compile-schemas $out/share/gsettings-schemas/${name}/glib-2.0/schemas
+    wrapProgram $out/libexec/tracker-writeback \
+      --prefix LD_LIBRARY_PATH : $out/lib/tracker-miners-2.0
```

Comment:
This is weird too. Why does meson not produce a correct rpath? Is there an upstream bug report?

Reply:
I'm not sure why, but it doesn't. I don't think there is an upstream bug report.

Reply:
It looks like `tracker-writeback` is missing `install_rpath`. Shall I open a bug?

Reply:
Bug: https://bugzilla.gnome.org/show_bug.cgi?id=795574
Patch: https://bugzilla.gnome.org/attachment.cgi?id=371423